scenarios/kubernetes_e2e.py

```diff
@@ -84,6 +84,12 @@ def add_files(self, env_files):
                         continue
                     self.env_files.append(self.parse_env(line))
 
+    def add_aws_cred(self, priv, pub, cred):
+        """Sets aws keys and credentials."""
+        self.add_environment('JENKINS_AWS_SSH_PRIVATE_KEY_FILE=%s' % priv)
+        self.add_environment('JENKINS_AWS_SSH_PUBLIC_KEY_FILE=%s' % pub)
+        self.add_environment('JENKINS_AWS_CREDENTIALS_FILE=%s' % path)
```

Comment:
do I want to copy the keys to ./~ssh in local mode?

I don't have a key so I don't know :imp: